<launch>
    <!-- Launch the hello_publisher node -->
    <node name="tello_driver" pkg="drone_swarm" type="tello_driver.py" output="screen"/>
      <arg name="name" default="tello" />
        <arg name="id" default="1" />
        <arg name="drone_ip" default="192.168.0.101" />
        <arg name="local_port" default="9010" />
        <arg name="group" default="A" />
        <arg name="target" default="[]" />   

</launch>

